pyvista/core/dataobject.py

```diff
@@ -145,6 +148,34 @@ def save(self, filename: str, binary=True, texture=None):
                 writer.SetEnableAlpha(True)
         writer.Write()
 
+    def _store_metadata(self):
+        """Store metadata as field data."""
+        fdata = self.field_data
+        for assoc_name in ('bitarray', 'complex'):
+            for assoc_type in ('POINT', 'CELL'):
+                assoc_data = getattr(self, f'_association_{assoc_name}_names')
+                if assoc_type in assoc_data:
+                    if assoc_data[assoc_type]:
+                        key = f'_PYVISTA_{assoc_name}_{assoc_type}_'.upper()
+                        fdata[key] = list(assoc_data[assoc_type])
```

Comment:
Why is this `list` call necessary?

Reply:
We can't assign lists to field data.

Should we enable this? Probably. This PR? Probably not.